install/windows/docker-ee.md

```diff
@@ -156,19 +157,14 @@ installs, or install on air-gapped systems.
 
 ## Install a specific version
 
-There are currently two channels available for Docker EE for Windows Server:
-
-* `18.09` - Use this version if you're using Docker Enterprise 2.1 (Engine, UCP, DTR), or if you're running Docker Engine alone. `18.09` is the default.
-* `17.06` - Use this version if you're using Docker Enterprise 2.0.
-
```

Comment:
@jasonbivins why did you cut this section?

Reply:
This was part of the section that was added to explain the 18.03 engine only release and even though it was updated I didn't think we needed it.  This is covered in the compatibility matrix anyway.

unless we're still selling 2.0 ? it should probably be re-added if we are...

Reply:
I'm going to leave this out for Windows. If we need to re-add it, please open a new issue.